README.md

```diff
@@ -845,10 +872,13 @@ Note that the transaction filter was factored out as it can be shared. The final
 
     });
 
-Before running this you should start with a clean ledger to avoid being confused by the unprocessed contracts from previous examples.
+Before running this you should start with a clean ledger to avoid being confused by the unprocessed contracts from previous examples:
```

Comment:
Maybe it's okay to make a more assertive statement like:
```
Before running the application, start with a clean ledger to avoid receiving unprocessed contracts from previous examples:
```

Reply:
Very good catch, thanks!

Addressed by https://github.com/digital-asset/ex-tutorial-nodejs/pull/7/commits/579220195e8ad201ccfe026a812abe5fd3d908d1

Reply:
Improved on with https://github.com/digital-asset/ex-tutorial-nodejs/pull/7/commits/7cbd34f6f64471bf6912614e32c97740d4aff4ac